AudioRecorder.py

```diff
@@ -102,6 +104,16 @@ def get_name(self):
         """Get the name of this device
         """
 
+    def enable(self):
```

Comment:
Although enable method is not used currently, I like that it is included. It will help with enabling, disabling functionality in the UI.